.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
 cache: pip
 python:
-  - 3.6-dev
+  - 3.6
+  - 3.7
   - 3.7-dev
+  - 3.8-dev
```

Comment:
Isn't there a way to make testing against 3.8 an allowed failure so it's more informational than anything?

Reply:
Thanks. I've made it so failures allowed in `3.8-dev` and `nightly`,